talent/engineering-teams-descr.md

```diff
@@ -0,0 +1,38 @@
+This document describes the engineering teams at source{d} and how they interact with each other.
+
+### Machine Learning Team
+
+- The Machine Learning team is focused on providing intelligent insights into nearly all the world's Open Source code. This includes typical Big Data analysis as well as solving sophisticated Machine Learning problems. The former is conducted on Apache Spark clusters, the latter leverages extensive GPGPU acceleration on custom hardware using deep learning techniques. Often ML problems we face are unsupervised and require novel research in the domain of ML on source code (MLoSC).
+
+### Applications Team
+
+- The Applications team applies the whole stack and expertise to solve real word problems as code duplication, smart diffs, search for repositories, projects evolving reports. This is done by implementing CLI/Web applications on top of the MLoSC research. Simple and intuitive applications are written mainly in Go and Scala, based on developer tooling prototypes.
+
+### Data Retrieval Team 
+
+- The Data Retrieval team is developing source{d}'s high-level Engine for running scalable data retrieval pipelines that process and manipulate any number of code repositories for source code analysis. Written mostly in Scala, it aims to be robust, friendly and flexible: it is built on top of Apache Spark, accessible both via Scala and Python Spark APIs, and capable of running on large-scale distributed clusters over petabytes of data. 
```

Comment:
go-git and git-base are not relevant to be mentioned?

Reply:
@ajnavarro I need your input here, as I'm not sure either. I guess they should be relevant.